src/components/content-tab-import-export.tsx

```diff
@@ -225,7 +225,9 @@ const ImportSite = ( {
 			<div className="a8c-subtitle-small mb-1">{ __( 'Import' ) }</div>
 			<div className="text-a8c-gray-70 a8c-body mb-4">
 				{ createInterpolateElement(
-					__( 'Import a Jetpack backup or a .sql database file. <button>Learn more</button>' ),
+					__(
+						'Import a Jetpack backup, another full-site backup, or a .sql database file. <button>Learn more</button>'
```

Comment:
```suggestion
						'Import a Jetpack backup, a full-site backup in another format, or a .sql database file. <button>Learn more</button>'
```

Reply:
Alternatively, we can also do something like:


```suggestion
						'Import a backup created with Jetpack, a full-site backup from another tool or service, or a .sql database file.. <button>Learn more</button>'
```